frontends/web/src/routes/account/account.tsx

```diff
@@ -84,17 +84,22 @@ export function Account({
   }, [code]);
 
   const maybeCheckBitsuranceStatus = useCallback(async () => {
-    if (account?.bitsuranceId) {
-      const insuredAccounts = await bitsuranceLookup(code);
+    if (account?.bitsuranceStatus) {
+      // we set the `persistCancel` flag to avoid notifying
+      // again in case of insurance deactivation.
```

Comment:
not sure I understand this comment, could you elaborate?

persistCancel = true leads to `backend.emitAccountsStatusChanged()` right? or do you mean some other notification?

Reply:
I mean the frontend notification dialog that is shown on the account page when the account insurance status changes from 'active' to something else (https://github.com/Beerosagos/bitbox-wallet-app/blob/bitsurance-dashboard-rebased/frontends/web/src/routes/account/account.tsx#L99)